src/planscape/impacts/views.py

```diff
@@ -150,3 +150,32 @@ def create(self, request, *args, **kwargs):
 
     def perform_create(self, serializer):
         return upsert_treatment_prescriptions(**serializer.validated_data)
+
+    # current implementation requires a tx_plan_id -- do we want to provide a non-nested option?
```

Comment:
@george-silva Should we do this in a completely non-nested way? In other words -- will there be a case when we don't have a treatmentplan id at all?